php-frontend/src/test/java/org/sonar/php/tree/symbols/ScopeTest.java

```diff
@@ -71,6 +71,15 @@ public void function_expression_scope() throws Exception {
     assertNotNull(functionExprScope.getSymbol("$e"));
   }
 
+  @Test
+  public void arrow_function_expression_scope() throws Exception {
+    Scope scope = getScopeFor(Kind.ARROW_FUNCTION_EXPRESSION);
+
+    assertThat(scope.getSymbols(Symbol.Kind.PARAMETER)).extracting(Symbol::name).containsExactly("$a");
+    assertThat(scope.getSymbols(Symbol.Kind.VARIABLE)).isEmpty();
+    assertThat(scope.getSymbol("$c", Symbol.Kind.VARIABLE)).isNotNull();
+  }
```

Comment:
it would be good to add test with shadowing of variables from outer scope

Reply:
I added a commit to test the shadowing